pkg/render/stats.go

```diff
@@ -0,0 +1,104 @@
+package render
+
+import (
+	"fmt"
+	"sort"
+
+	"github.com/chainguard-dev/bincapz/pkg/bincapz"
+	"github.com/chainguard-dev/bincapz/pkg/report"
+)
+
+func riskLevelStatistics(files map[string]bincapz.FileReport, riskMap map[int][]string, riskPercentages map[int]float64) []bincapz.Kv {
+	for path, rf := range files {
+		riskMap[rf.RiskScore] = append(riskMap[rf.RiskScore], path)
+		for riskLevel := range riskMap {
+			riskPercentages[riskLevel] = (float64(len(riskMap[riskLevel])) / float64(len(files))) * 100
+		}
+	}
+
+	var riskLevelStatistics []bincapz.Kv
+	for k, v := range riskPercentages {
+		riskLevelStatistics = append(riskLevelStatistics, bincapz.Kv{Key: k, Value: v})
+	}
+	sort.Slice(riskLevelStatistics, func(i, j int) bool {
+		return riskLevelStatistics[i].Value > riskLevelStatistics[j].Value
+	})
+
+	return riskLevelStatistics
+}
+
+func packageRiskStatistics(files map[string]bincapz.FileReport, packageRiskMap map[string]int) ([]bincapz.KvStr, int) {
+	var numNamespaces int
+	packageRiskPercentages := make(map[string]float64)
```

Comment:
Same here, this could easily just be "percentages" or "stats"

Reply:
Addressed in [`5ad17ab` (#121)](https://github.com/chainguard-dev/bincapz/pull/121/commits/5ad17abd03cd920455ed146562cb0bf3c9ac992f).